File: ace75/isSubsequence.py
def isSubsequence(s:str,t:str) -> str:
	s_pointer = 0
	t_pointer = 0
	while s_pointer<len(s) and t_pointer<len(t):
		if s[s_pointer] == t[t_pointer]:
			s_pointer+=1
		t_pointer+=1
	return s_pointer == len(s)
	

File: ace75/isSubsequence.cpp
#include<bits/stdc++.h>
class Solution {
	bool isSubsequence(std::string s,std::string t){
		int n = s.length(),m=t.length();
		int j=0;
		for(int i=0;i<m and j<n;i++){
			if(s[j]==t[i]){
				j++;
			}
		}
		return(j==n);
	}
}